src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx

```diff
@@ -150,3 +154,19 @@ class OrganizationProjects extends AsyncView {
 }
 
 export default withOrganization(OrganizationProjects);
+
+const GridPanelItem = styled(PanelItem)`
+  display: flex;
+  align-items: center;
+  padding: 0;
+`;
+
+const ContainerProjectListItem = styled('div')`
+  padding: ${space(2)};
+  flex: 1;
+`;
+
+const ContainerProjectStatsGraph = styled('div')`
```

Comment:
nit: We generally use `Container` as a suffix, also `Wrapper` might be better not to be confused with our "Container" components that do data fetching, etc.